### PR TITLE
Paths fixer

### PR DESCRIPTION
**Pull request by apurvann**
Minor changes, mostly with the structure of the project. I forgot to create the pull request while testing swift. So doing it now. Travis config was also changed because build was failing due to new structure.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,3 +1,99 @@
...
+    # https://packaging.python.org/en/latest/single_source_version.html
+    version='1.2.0',
+
+    description='A sample Python project',
+
+    # The project's main homepage.
+    url='https://github.com/pypa/sampleproject',
+
+    # Author details
+    author='The Python Packaging Authority',
```
Comment (hhamalai) [inline setup.py]: Change actual author

File: setup.py [hunk 2]
```diff
@@ -1,3 +1,99 @@
...
+    author_email='pypa-dev@googlegroups.com',
```
Comment (hhamalai) [inline setup.py]: Change actual author email

File: setup.py [hunk 3]
```diff
@@ -1,3 +1,99 @@
...
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.2',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+    ],
+
+    # What does your project relate to?
+    keywords='sample setuptools development',
```
Comment (hhamalai) [inline setup.py]: use proper keywords

File: sparkles/modules/utils/runner.py [hunk 4]
```diff
@@ -56,21 +61,23 @@ def run_analysis(self, modulename, params, inputs):
 
         am = self.session.query(Analysis).from_statement(text("SELECT * FROM analysis where name=:name")).\
             params(name=modulename).first()
-
+        print(am.filepath)
```
Comment (hhamalai) [inline sparkles/modules/utils/runner.py]: remove or use logging

**Comment by hhamalai** [comment]
if modules directory is removed, it should be removed from Travis configuration's excludes

**Comment by hhamalai** [comment]
use uniform naming for variable, either camelCase or underscores

**Comment by apurvann** [comment]
Now it is fixed and ready to be tested. Please merge after review.

**Pull request merged**